suricata.yaml.in

```diff
@@ -555,6 +555,10 @@ logging:
   # This value is overridden by the SC_LOG_OP_FILTER env var.
   default-output-filter:
 
+  # Requires --enable-libunwind. Display a brief diagnostic message with the
+  # offending stacktrace when enabled _and_ configured.
+  #sigsegv-stacktrace: off
```

Comment:
I wonder if someone has gone through the trouble of adding `--enable-libunwind` if it should be enabled by default?

Reply:
Good point. We can flip this setting so the default value is `on`